src/test/resources/core-site.xml

```diff
@@ -1,61 +1,69 @@
 <configuration>
-<!-- use this only if it does not affect your hadoop site. For standalone spark
-deployments this is useful as you can use relative paths eg ./bin/dl -ls /
--->
-<property>
-  <name>fs.defaultFS</name>
-  <value>dl://node10930-datanodes-data-lake01-uk-reading.bigstep.io:14000/data_lake/dl459</value>
-</property>
-
-
-<property>
-  <name>fs.dl.impl</name>
-  <value>com.bigstep.datalake.DLFileSystem</value>
-</property>
-
-<property>
-  <name>fs.dl.impl.kerberosPrincipal</name>
-  <value>k7@bigstep.io</value>
-</property>
-
-<property>
-  <name>fs.dl.impl.kerberosKeytab</name>
-  <value>/etc/k7.keytab</value>
-</property>
-
-<property>
-  <name>fs.dl.impl.kerberosRealm</name>
-  <value>bigstep.io</value>
-</property>
-
-<property>
-  <name>fs.dl.impl.defaultEndpoint</name>
-  <value>node10930-datanodes-data-lake01-uk-reading.bigstep.io:14000</value>
-</property>
-
-<property>
-  <name>fs.dl.impl.homeDirectory</name>
-  <value>/data_lake/dl459</value>
-</property>
-
-<!-- optional -->
-<property>
-  <name>fs.dl.impl.defaultFilePermissions</name>
-  <value>00640</value>
-</property>
-
-<!-- optional. This is the umask to be enforced on the permissions being
-deployed in the datalake. As a security precaution, to prevent accidentally
-making the files publicly available, te default mask (007) overwrites all the
-'others' settings. -->
-<property>
-  <name>fs.dl.impl.defaultUMask</name>
-  <value>007</value>
-</property>
-
-<!-- This is provided to allow the interaction with a generic httpfs server for testing -->
-<property>
-  <name>fs.dl.impl.transportScheme</name>
-  <value>https</value>
-</property>
+  <!-- use this only if it does not affect your hadoop site. For standalone spark
```

Comment:
I think you should add this file to gitignore. Every developer should configure their own. Maybe simply put the sample core-site.xml here as a placeholder.